src/main/java/com/google/devtools/build/lib/packages/TargetUtils.java

```diff
@@ -59,6 +62,17 @@ public boolean apply(String tag) {
     }
   };
 
+  // Not all the target level tags are propagated as execution requirements to the actions, because
+  // we do not want to propagate custom user's tags or create potential conflicts
+  // with execution requirements declared on the rules.
+  // See https://github.com/bazelbuild/bazel/issues/7766 for details.
+  private static final Predicate<String> TAGS_PROPAGATED_TO_EXEC_INFO =
+      tag -> tag.equals("no-remote")
+          || tag.equals("no-cache")
+          || tag.equals("no-sandbox")
+          || tag.equals("no-remote-exec")
+          || tag.equals("no-remote-cache");
```

Comment:
I also can't find any reference to those two tags having special meaning here: https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes

Reply:
Ah they're planned tags for more control over remote caching: https://github.com/bazelbuild/bazel/issues/7932#issuecomment-495173015

Reply:
Yes, those are only planned, but I decided to add them to the whitelist from the beginning. I'll think about it, perhaps to avoid confusions it would be better to not include them here and just add a comment for the future task #7932, where they will be introduced.

Reply:
done